docs/example-react-router.mdx

```diff
@@ -133,8 +133,35 @@ test('rendering a component that uses useLocation', () => {
 
 ## Testing Library and React Router v5
 
-In React Router v5, you need to pass the history object as a whole to the Route
-component.
+Given your App uses the correct React Router v5 API:
+
+```jsx
+import { Link, Route, Switch, useLocation } from 'react-router-dom'
+
+// ...
```

Comment:
Any reason to remove the `LocationDisplay` and page components? We want an example that will work in a copy paste scenario :)

Reply:
No special reason, I was just thinking that we don't need to repeat the same information, but if we want to let user copy-paste then I'm happy to add the rest of the code.

Reply:
I think we're good with having these kind of duplications if it will make people's life a bit easier, thanks!
You can publish this PR btw.

Reply:
@MatanBobi I have updated as per you feedback. Sorry for the force push, I realized the author email was wrong so I have to amend the commits

Reply:
No worries, thanks :)